runtime/westend/src/lib.rs

```diff
@@ -1105,10 +1105,23 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPallets,
+	SetStakingLimits,
 >;
 /// The payload being signed in transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
+pub struct SetStakingLimits;
+impl frame_support::traits::OnRuntimeUpgrade for SetStakingLimits {
+	fn on_runtime_upgrade() -> Weight {
+		<pallet_staking::MinNominatorBond<Runtime>>::put(1 * UNITS);
+		<pallet_staking::MaxNominatorsCount<Runtime>>::put(1000);
+		<pallet_staking::MinValidatorBond<Runtime>>::put(10 * UNITS);
+		<pallet_staking::MaxValidatorsCount<Runtime>>::put(10);
```

Comment:
We want to make westend more complex than polkadot in order to detect issues in the future.

We should have no limits here

Reply:
I preferred some limits here, and I set them intentionally low, such that we can try chilling others there.